src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableCommonContext.cs

```diff
@@ -136,6 +136,11 @@ Guid IDeterministicExecutionContext.NewGuid()
         void IDeterministicExecutionContext.SignalEntity(EntityId entity, string operationName, object operationInput)
         {
             this.ThrowIfInvalidAccess();
+            if (operationName == null)
```

Comment:
Wouldn't this allow for empty or whitespace strings to be used as operations?

Reply:
Yes, the idea was to now allow empty strings as operation names.

It makes sense for REST API, and I did not see a strong reason to disallow it in general.